nvd3/linePlusBarChart.py

```diff
@@ -104,17 +104,22 @@ def __init__(self, **kwargs):
                                         "function(d) { return d3.format(',f')(d) }")
         self.yaxis2_format = kwargs.get('yaxis2_format',
                                         "function(d) { return d3.format(',f')(d) }")
-
+        # kwargs.get('x_is_date', True) means :add key('x_is_date') in dict,or/and set a defualt value while it=None.return the value after modify
```

Comment:
In fact, most of "code commented" was just mention myself where I modify in case error occurring, for our software project of stock :p
also some of my "code commented" just a discovery in python's grammar that confused me , because I am new at PY.
My friend who teach me about Github's usage, is also thinks those "code commented" is somewhat needless (and ridiculous) , so you can just delete it. I regret forgetting doing this job before uploading.

At 2015-08-16 21:31:26, "Areski Belaid" <notifications@github.com> wrote:


In nvd3/linePlusBarChart.py:

> @@ -104,17 +104,22 @@ def __init__(self, **kwargs):
>                                          "function(d) { return d3.format(',f')(d) }")
>          self.yaxis2_format = kwargs.get('yaxis2_format',
>                                          "function(d) { return d3.format(',f')(d) }")
> -
> +        # kwargs.get('x_is_date', True) means :add key('x_is_date') in dict,or/and set a defualt value while it=None.return the value after modify


adding "code commented" in it s not necessary in a PR

—
Reply to this email directly or view it on GitHub.